embeddings/pipeline/lightning_classification.py

```diff
@@ -18,12 +18,6 @@
 class LightningClassificationPipeline(
     LightningPipeline[datasets.DatasetDict, Dict[str, nptyping.NDArray[Any]], Dict[str, Any]]
 ):
-    DEFAULT_TASK_TRAIN_KWARGS = {"devices": "auto", "accelerator": "auto"}
```

Comment:
Why it is removed in that PR?

Reply:
It was moved to the parent `LightningPipeline`, due to duplicated code in it's children.

Reply:
Ok. There will be not so easy rebase with conig's pr unfortunately :(